### PR TITLE
feat(channels): suggested nodes on channel form

### LINKED ISSUE #1615: I should see a list of suggested nodes on the add channel form so that it's easy for me to open a channel with popular routing nodes
## Detailed Description

Add suggested nodes view to the bottom of the node search form.

Should show under the search bar when the search bar is empty.

Should be replaced with search results when the user starts typing.

image

## Context

New Channel UI

### PR DESCRIPTION
**Pull request by mrfelton**
## Description:

Add suggested nodes view on the channel create page if the user has not yet typed anything in the search box.

## Motivation and Context:

Resolves #1615

## How Has This Been Tested?

Manually

## Screenshots (if appropriate):
image

## Types of changes:

New Feature

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have reviewed and updated the documentation accordingly.
- [x] I have read the _CONTRIBUTING_ document.
- [ ] I have added tests to cover my changes where needed.
- [ ] All new and existing tests passed.
- [x] My commits have been squashed into a concise set of changes.

### CODE REVIEW

**Comment by mrfelton** [comment]
@JimmyMow I made some updates to the layout based on our discussions in Slack.

Could you host the images on your domain and update the suggested nodes list to reference them from there please?

File: app/components/Channels/ChannelNodeSearch.js [hunk 1]
```diff
@@ -61,7 +62,8 @@ class ChannelNodeSearch extends React.PureComponent {
    * Handle connect to node click.
    */
   handleClickConnect = node => {
-    this.formApi.setValue('search', `${node.pub_key}@${node.addresses[0].addr}`)
+    const { updateContactFormSearchQuery } = this.props
+    updateContactFormSearchQuery(`${node.pub_key}@${node.addresses[0].addr}`)
```
Comment (korhaliv) [inline app/components/Channels/ChannelNodeSearch.js]: Could you please add a debounce out of the box?

File: app/components/Channels/ChannelCreate.js [hunk 2]
```diff
@@ -19,10 +23,30 @@ class ChannelCreate extends React.Component {
...
+        <Panel {...rest}>
+          <Flex justifyContent="center">
+            <Panel.Header width={9 / 16}>
+              <ChannelCreateHeader />
+              <Bar pt={2} mb={3} />
+            </Panel.Header>
+          </Flex>
+
+          <Panel.Body css={{ 'overflow-y': 'overlay', 'overflow-x': 'hidden' }}>
+            <Flex flexDirection="Column" alignItems="center" css={{ height: '100%' }}>
```
Comment (korhaliv) [inline app/components/Channels/ChannelCreate.js]: typo `flexDirection="Column" `

File: app/components/Channels/ChannelNodeSearch.js (hunk 1 above)
Comment (mrfelton) [reply app/components/Channels/ChannelNodeSearch.js]: I deliberately didn't add a debounce here, as it just makes the UI unnecessarily unresponsive. This is the `Connect` button. When the user clicks on the button the app should respond instantly, not in 300 ms.

File: app/components/Channels/ChannelCreate.js (hunk 2 above)
Comment (mrfelton) [reply app/components/Channels/ChannelCreate.js]: Updated

File: app/components/Channels/ChannelNodeSearch.js (hunk 1 above)
Comment (korhaliv) [reply app/components/Channels/ChannelNodeSearch.js]: Sorry, it's ok. Haven't seen a method name. Search is already debounced here

**Pull request merged**